internal/cli/doc-gen.go

```diff
@@ -120,6 +131,21 @@ layout: default
 `
```

Comment:
**Request:** Not necessarily in the purview of this PR, but I've noticed the lack of space in the "InheritedFlags" H2. Can you make it into "Inherited Flags"? Normally, I wouldn't request an out-of-scope change but this PR is already fairly large and rebuilds the docs anyway.

Reply:
Done in 87eada9e4273063d1ce9051aef2fc3a963d631ba